pkg/kube/inject/testdata/inject/auth.non-default-service-account.yaml.injected

```diff
@@ -17,7 +17,11 @@ spec:
         prometheus.io/path: /stats/prometheus
         prometheus.io/port: "15020"
         prometheus.io/scrape: "true"
+        sidecar.istio.io/interceptionMode: REDIRECT
         sidecar.istio.io/status: '{"version":"","initContainers":["istio-init"],"containers":["istio-proxy"],"volumes":["istio-envoy","istio-data","istio-podinfo","istio-token","istiod-ca-cert"],"imagePullSecrets":null}'
+        traffic.sidecar.istio.io/excludeInboundPorts: "15020"
```

Comment:
In the case of CNI, it actually injects all of those in all situations. See https://github.com/istio/istio/blob/82ace29a6009e40d0eb68e81ffa41c891a8acc94/cni/cmd/istio-cni/redirect.go#L247.

This was done because these annotations were missing and customers were hitting situations where setting the excludeInboundPorts annotation overwrote the defaults and broke some traffic.